From: Skipper Seabold <jsseabold@gmail.com>
Subject: MAINT: Use ip_directive shipped with IPython. Closes #1428.

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -32,8 +32,8 @@ extensions = ['sphinx.ext.autodoc', 'sph
               'sphinx.ext.inheritance_diagram',
               'matplotlib.sphinxext.plot_directive',
               'matplotlib.sphinxext.only_directives',
-              'ipython_console_highlighting',
-              'ipython_directive',
+              'IPython.sphinxext.ipython_console_highlighting',
+              'IPython.sphinxext.ipython_directive',
               'numpy_ext.numpydoc',
               'github' # for GitHub links
               ]
